### PR TITLE
Add release post for Podman 1.2.0

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

File: _posts/2019-04-10-podman-release-v1.2.0.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+![podman logo](https://podman.io/images/podman.svg)
+
+## Welcome to Podman 1.2.0!
+
+Podman 1.2.0 has been released, featuring many exciting new features and fixes for numerous bugs. With 1.2.0, Podman added support for container healthchecks, an events system, and a way to view image layers as a tree. Over 30 bugs were fixed in this new release, including numerous issues with rootless Podman. We also upgraded the version of Buildah driving `podman build` from v1.7 to v1.7.2, picking up numerous fixes.
+
+<!--readmore-->
+
+Our new Podman release includes support for container healthchecks. Healthchecks provide additional information on container status, running image-defined checks to verify that the application in a container is working properly. Any containers with healthchecks defined will run them automatically, and their status can be checked with `podman inspect`. The `podman healthcheck run` command can also be used to manually trigger a healthcheck.
```
Comment (baude) [inline _posts/2019-04-10-podman-release-v1.2.0.md]: running image or user-defined checks

File: _posts/2019-04-10-podman-release-v1.2.0.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Podman also added a new command, `podman events`, that can be used to view major lifecycle events for containers, pods, and images as they occur. This command and its corresponding Varlink API can be used by tools which wish to check the overall status of the system, or check when a specific container starts or exits.
```
Comment (baude) [inline _posts/2019-04-10-podman-release-v1.2.0.md]: maybe provide an example of an event or two?

File: _posts/2019-04-10-podman-release-v1.2.0.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+The `podman image tree` command was also added. This command will print a tree representation of an image's layers. This can be used to easily identify an image's dependencies.
```
Comment (baude) [inline _posts/2019-04-10-podman-release-v1.2.0.md]: example?

**Comment by TomSweeneyRedHat** [comment]
LGTM once @baude's comments are addressed.

**Comment by mheon** [comment]
Updated. Would love to have other people check how the examples actually look in-page - I can't decide if I like them or not. We could potentially do images instead.

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM, including the examples.
@rhatdan @baude WDYT?

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**